examples/agones-game-controller/main.tf

```diff
@@ -85,15 +70,26 @@ module "eks_blueprints_kubernetes_addons" {
   eks_oidc_provider    = module.eks.oidc_provider
   eks_cluster_version  = module.eks.cluster_version
 
-  # EKS Managed Add-ons
-  enable_amazon_eks_vpc_cni    = true
-  enable_amazon_eks_coredns    = true
-  enable_amazon_eks_kube_proxy = true
-
   # Add-ons
-  enable_metrics_server         = true
-  enable_cluster_autoscaler     = true
-  enable_aws_cloudwatch_metrics = true
+  enable_metrics_server     = true
+  enable_cluster_autoscaler = true
+
+  # NOTE: Agones requires a Node group in Public Subnets and enable Public IP
+  enable_agones = true
```

Comment:
I don't know why GitHub does this with diffs - the `upstream-with-k8s-addons` was deleted, not renamed